tests/templates/kuttl/kerberos/12-rego-rules.txt.j2

```diff
@@ -183,6 +219,8 @@ data:
         "bob/access-hdfs.$NAMESPACE.svc.cluster.local@{{ test_scenario['values']['kerberos-realm'] }}": []
     }
 
+    groups_for_short_user_name := {}
```

Comment:
I assume this is empty on purpose (even though it is used in the rules above).

Reply:
Exactly! We only map groups based on the long username (`alice/access-hdfs.$NAMESPACE.svc.cluster.local@{{ test_scenario['values']['kerberos-realm'] }}`) instead of only `alice` for security reasons here